astropy/cosmology/parameter.py

```diff
@@ -210,8 +212,104 @@ def register(fvalidate):
 
     # -------------------------------------------
 
+    def _get_init_arguments(self, repr=True):
```

Comment:
How about instead calling the argument `interpreted=False`? Since it is not only `__repr__` but also `clone()` that wants the non-interpreted version.